src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -142,6 +144,7 @@ const ContextualMenu = <L,>({
   const labelNode = toggleLabel ? <span>{toggleLabel}</span> : null;
   const wrapperClass = classNames(className, "p-contextual-menu", {
     [`p-contextual-menu--${adjustedPosition}`]: adjustedPosition !== "right",
+    "is-dark": dark,
```

Comment:
Does Vanilla support putting `is-dark` on the body tag and have it cascade all the way down? 

If not, while this may be a one-off just to have a dark dropdown and not a 'darkmode' component, if we have to add a prop to every component to turn it into darkmode that's not going to be very scalable.

Reply:
I added it because I've seen it being a prop on other component. `Strip` it seems has it.

At this point themes in Vanilla are set on component level. It seems to be a design decision that not every component is light or dark.
For example, on most our sites, top navigation uses dark theme while rest of the page is light theme. We generally set this on component level, either as settings in SCSS or via `is-dark` class name.

But, your comment made me think, and I guess that you may be right that probably we don't want to directly reimplement the same approach here in react components.

If any react project wants to change the theme of components they can (and should) do it via scss settings for whole project. Let's not provide it as a props on individual components until we really think that's the way to go forward.